chrome/manifest.json

```diff
@@ -31,7 +31,6 @@
 			"https://*.reddit.com/*"
 		],
 		"exclude_matches": [
-			"https://mod.reddit.com/*",
```

Comment:
Can you update this change across all browser manifests?
Edge - https://github.com/honestbleeps/Reddit-Enhancement-Suite/blob/master/edge/manifest.json
Firefox - https://github.com/honestbleeps/Reddit-Enhancement-Suite/blob/master/firefox/manifest.json